### ISSUE TITLE
Is there a way to make it work without deleting the data from Adapter?

### ISSUE THREAD

**Issue by zaur** [Diolor/Swipecards#58]
I'm trying to use SwipeCards with CursorAdapter. What I wanna achieve is to iterate through items in cursor instead of deleting swiped item from adapter. Any ideas/suggestions?

**Comment by Diolor** [comment]
Good question. I cannot think at this point with the current architecture to do so.

**Comment by lolobosse** [comment]
I got the same issue as @zaur and I solved it by proxying the Cursor in an array. This is a decent solution but it doesn't really answer the question...

(Just saying for people having the same issue :wink: ).

I tried with `requery()` and timestamps in the DB but it is a real mess...

**Comment by rashikaranpuria** [comment]
Before actually trying, does this library work with CursorAdapter ?

**Comment by vishalshah133** [comment]
Yes. It does work. But it stops to swipe after the 1st card swipe. Any workarounds @Diolor ?

**Comment by lolobosse** [comment]
@vishalshah133 So no, it doesn't work 😆
